### PR TITLE
Add setInputFiles local file example

### PR DESCRIPTION
**Pull request by ankur22**
## What?

This adds an example to `setInputFiles` which reads a file from the local filesystem.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

https://github.com/grafana/k6-docs/pull/1528#issuecomment-2013662608

### CODE REVIEW

File: docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default async function () {
   const page = browser.newPage();
 
   // In this example we create a simple web page with an upload input field.
+  // Usually you would use page.goto to navigate to a page with an file input field.
```
Comment (inancgumus) [inline docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md:47]: There are others with this typo in other places.

```suggestion
  // Usually, you would use page.goto to navigate to a page with a file input field.
```

File: docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md [hunk 2]
```diff
@@ -70,3 +71,80 @@ export default async function () {
...
+  },
+};
+
+// Declare the location of the file on the local filesystem.
+let file;
+(async function () {
+  file = await open('/abs/path/to/file.txt');
+})();
+
+export default async function () {
```
Comment (inancgumus) [inline docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md:103]: Should we use `try` `catch`?

**Review by inancgumus — APPROVED** [review]
Looks good 👍

File: docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md (hunk 2 above)
Comment (inancgumus) [reply docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md:103]: Yes, to be consistent, it makes. And, yep, I know what you mean and agree it's worth discussing. I created an issue about this behavior once: https://github.com/grafana/xk6-browser/issues/814.

Comment (ankur22) [reply docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md:105]: Changed in 74ac66c183c4e84edd044d6fdbc036f68b974bdc

**Review by heitortsergent — APPROVED** [review]
LGTM, thanks for adding the example so quickly! Also, tagging @tom-miseur here if he wants to take a look, he and I had been chatting about adding more examples to the docs.

**Comment by ankur22** [comment]
@heitortsergent I'm going to merge this in, but happy to open a new PR and iterate on it further if needed.

**Pull request merged**